cea/default.config

```diff
@@ -140,7 +140,7 @@ buildings.category = Advanced
 
 loads-output =
 loads-output.type = MultiChoiceParameter
-loads-output.choices = QEf, QHf, QCf, Ef, Qhsf, Qhs, Qhsf_lat, Qwwf, Qww, Qcsf, Qcs, Qcsf_lat, Qcdataf, Qcref, Qhprof, Edataf, Ealf, Eaf, Elf, Eref, Eauxf, Eauxf_ve, Eauxf_hs, Eauxf_cs, Eauxf_ww, Eauxf_fw, Eprof, Ecaf, Egenf_cs, Qgain_light, Qgain_app, Qgain_pers, Qgain_data, Qgain_base, Qgain_roof, Qgain_vent, Qgain_wall, Qgain_wind, I_sol, I_rad
+loads-output.choices = QEf, QHf, QCf, Ef, Egenf_cs, Qhs_sen_shu, Qhs_sen_ahu, Qhs_lat_ahu, Qhs_sen_aru, Qhs_lat_aru, Qhs_sen_sys, Qhs_lat_sys, Qhs_em_ls, Qhs_dis_ls, Qhs, Qhsf, Qhsf_lat, Qwwf, Qww, Qhsf_shu, Qhsf_ahu, Qhsf_aru, Qcdataf, Qcref, Qcs_sen_scu, Qcs_sen_ahu, Qcsf_scu, Qcsf_ahu, Qcsf_aru, Qcs_lat_ahu, Qcs_sen_aru, Qcs_lat_aru, Qcs_sen_sys, Qcs_lat_sys, Qcs_em_ls, Qcs_dis_ls, Qcsf, Qcs, Qcsf_lat, Qhprof, Edataf, Ealf, Eaf, Elf, Eref, Eauxf, Eauxf_ve, Eauxf_hs, Eauxf_cs, Eauxf_ww, Eauxf_fw, Eprof, Ecaf, Egenf_cs, Q_gain_sen_app, Q_gain_sen_light, Q_gain_sen_peop, Q_gain_sen_data, Q_loss_sen_ref, Q_gain_sen_env, Q_gain_sen_vent, Q_gain_sen_wind, I_sol, I_rad, Q_gain_lat_peop, mcpwwf, mcpdataf, mcpref, mcptw, mcpcsf_ahu, mcpcsf_aru, mcpcsf_scu, mcphsf_ahu, mcphsf_aru, mcphsf_shu, mcpcsf, mcphsf, Twwf_sup, T_int, T_ext, Twwf_re, Tcdataf_re, Tcdataf_sup, Tcref_re, Tcref_sup, Tcsf_re_ahu, Tcsf_re_aru, Tcsf_re_scu, Tcsf_sup_ahu, Tcsf_sup_aru, Tcsf_sup_scu, Thsf_re_ahu, Thsf_re_aru, Thsf_re_shu, Thsf_sup_ahu, Thsf_sup_aru, Thsf_sup_shu, Thsf_sup, Tcsf_sup, Thsf_re, Tcsf_re, Name, Af_m2, Aroof_m2, GFA_m2, people0
 loads-output.help = a list of loads to run the simulation for - leave blank to simulate all loads in demand_writers
```

Comment:
@gabriel-happle actually, there is one input variable for temperatures and massflows called temperature-output, massflows-output. Please use those to indicate which temperatures and massflows to output. 

No need to include Name, Af_m2, Aroof_m2, GFA_m2, people0 as tehy are always by default part of the output.